### DISCUSSION TITLE
这件事情应该会影响到人们买房的热情吧

### DISCUSSION THREAD

**Discussion by summerness** [WeNeedHome/SummaryOfLoanSuspension#429] [General]
也许事情会朝着 没人买房 --> 没有房企拿地 --> 地方政府收入下降 --> ....
如果这样，最终会影响到每一个人
希望顺利解决

**Comment by ghost** [comment]
这个时候接盘，祝你好运

**Comment by yb3616** [comment]
其实这种事情，打掉几只老虎就好了，钱在哪他们比谁都清楚

**Reply by summerness** [reply]
地方政府现在是真没钱了，之前做某个项目，xx城市的xx部门负责人告诉我们，账上就只剩5w了

**Reply by zhw173** [reply]
要真敢对老虎下手早下手了，也许老虎比政府权力大吧

**Comment by nekomatawj** [comment]
现在买房不是把人往火坑里推？

**Comment by yuntun7zz** [comment]
没有谁愿意把自己后面30年卖给别人，又何来热情一说？

**Reply by summerness** [reply]
前几年大家不都是有点钱都跑去买房

**Reply by aabbcczzz** [reply]
> 前几年你们不是有点钱都跑去买房了

之前有涨价预期，买了就是赚，现在没有了。又或者新楼盘更稀缺，二手房上涨也有可能。

**Comment by LeeNNN** [comment]
放心吧，12号南京土拍成交价727亿全国第一

**Reply by hasXSS** [reply]
民营房企成交57亿, 都是国企在拿地, 左手交右手

**Reply by BlackBerryPi** [reply]
现在土拍入局的都是卖给城投公司，还是要从财政里出，本身城投公司债务就很重了，这样只会雪上加霜，迟早会遍地流拍

**Reply by chaolia** [reply]
我愿称此为“武当梯云纵”

**Comment by xiejun946** [comment]
政府收入结构应该修改，增加财富税，高收入人群收取百分之50的税，个税收几个钱，千把亿而已。还有全国这么多个体户不交税的，网红的，明星的。房地产正常流转，下游供应商垮不了

**Comment by yuntun7zz** [comment]
买房何来热情一说？如果不是后半生的利益和房子高度绑定之外，谁愿意把自己后面30年卖了？

**Comment by kennard520** [comment]
如果没有高额房贷，大家或许会过的很幸福，中国人民的幸福指数还不如印度，如果没有高额房贷，或许，我还是喜欢中国
